pom.xml

```diff
@@ -124,7 +124,6 @@
             <groupId>com.google.code.findbugs</groupId>
             <artifactId>jsr305</artifactId>
             <version>3.0.2</version>
-            <scope>test</scope>
```

Comment:
This adds jsr305 to the runtime dependencies of the library. I think that's a high price for one usage of the `@Nullable` annotation. I suggest keeping the dependency in `test` scope and removing the annotation usage in `JadConfig`.

Reply:
Ok, makes sense, removed and returned to test scope only.